core/templates/pages/admin-page/activities-tab/admin-dev-mode-activities-tab.component.html

```diff
@@ -132,6 +132,28 @@ <h3>Generate a dummy classroom</h3>
       </div>
     </div>
   </mat-card>
+  <mat-card class="oppia-page-card oppia-long-text oppia-long-text-card">
```

Comment:
This only shows in dev mode. Needs to be in "misc" tab along with other admin page utilities so it can be run on prod.

Reply:
Resolving this conversation as the code is not present anymore.